src/i_oalsound.c

```diff
@@ -490,49 +491,148 @@ boolean I_OAL_AllowReinitSound(void)
     return (alcIsExtensionPresent(oal->device, "ALC_SOFT_HRTF") == ALC_TRUE);
 }
 
-boolean I_OAL_CacheSound(ALuint *buffer, ALenum format, const byte *data,
-                         ALsizei size, ALsizei freq)
+#define SOUNDHDRSIZE 8
+#define SOUNDPADSIZE 16
```

Comment:
Maybe we can use this opportunity to rename these to something more descriptive since they only apply to DMX sounds  (`DMXHDRSIZE` and `DMXPADSIZE` ?) and move them to the top of this file.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1141/commits/49625469297cda09cdcb2b46373a6c28cfaba21b